test/time_test.py

```diff
@@ -53,23 +67,36 @@ def test_get_rawtime(self):
             time.sleep(delay)
             c.tick(framerate_limit)
             c1 = c.get_rawtime()
-            self.assertAlmostEqual(delay_miliseconds, c1, delta=delta)
-
-       #Testing get_rawtime() = get_time()
+            if not _is_almost_equal(delay_miliseconds, c1, delta):
+                errcnt += 1
+        
+        if errcnt >= NUM_TRIES:
+            self.fail("error while doing get_rawtime()")
+            
+        errcnt = 0
+        
+        # Testing get_rawtime() = get_time()
         for f in range(iterations):
             time.sleep(delay)
             c.tick()
             c1 = c.get_rawtime()
             c2 = c.get_time()
-            self.assertAlmostEqual(c1, c2, delta=delta)
+            
+            self.assertEqual(c1, c2)
+            if not _is_almost_equal(delay_miliseconds, c1, delta):
+                errcnt += 1
+        
+        if errcnt >= NUM_TRIES:
+            self.fail("error while doing get_rawtime()")
 
 
     def test_get_time(self):
-        #Testing parameters
+        # Testing parameters
         delay = 0.1 #seconds
-        delay_miliseconds = delay*(10**3)
-        iterations = 10
-        delta = 50 #milliseconds
+        delay_milliseconds = delay * 1000
+        iterations = 5
+        delta = 0.1
+        errcnt = 0
 
         #Testing Clock Initialization
```

Comment:
should be a space after the # here and two before it. Looks like it is an old error in the test, but we may as well clean house in here with all the changes.

Reply:
Done :)